dist.ini

```diff
@@ -0,0 +1,25 @@
+name     = UUID4-Tiny
+author   = CV-Library <cvlibrary@cpan.org>
+license  = Perl_5
+copyright_holder = CV-Library
+copyright_year   = 2019
+
+[AutoPrereqs]
+[MakeMaker]
+[AssertOS]
+os = Linux
```

Comment:
is there a way we can limit it to AMD64? Since that's the type of syscall we're using.

Reply:
Doesn't seem to be possible in AssertOS itself. Any known alternatives out there?

Reply:
Something like a Makefile check for
```
$ perl -E 'say 8 == length pack "P"'
1
```
Is the best I've got, any ideas @jjatria ?

Or we could use the right syscall number on 32bit if anyone has a box to test on? I don't suppose Travis supports 32bit?

Reply:
I could remember doing this in a personal project once, but when I checked how I did, it was basically

```
$ perl -E 'say 1 if `uname -a` =~ /x86_64/'
1
```

so I'm open to alternatives :sweat_smile:

Reply:
We can go [really magic](https://stackoverflow.com/a/49368448/807650): `log(~0 +1)/log(2)`

Reply:
Went with something similar to the `length pack`.